### PR TITLE
Parsing-a-mime-type do not check for the condition if sequence[s] is undefined

### PR DESCRIPTION
**Pull request by Ritsyy**
Fix https://www.w3.org/Bugs/Public/show_bug.cgi?id=29243.

### CODE REVIEW

**Comment by Ritsyy** [comment]
@annevk if you could review it, thank you.

File: Overview.src.html [hunk 1]
```diff
@@ -279,6 +279,9 @@ <h2 class="no-num no-toc">Table of contents</h2>
    Initialize <var>parameters</var> to the empty dictionary ({}).
 
   <li>
+     If <var>sequence</var>[<var>s</var>] is undefined, return undefined.
```
Comment (annevk) [inline Overview.src.html]: This seems to be the wrong place for this. It needs to happen as the first step of the while loop per the bug report, no?

Comment (GPHemsley) [inline Overview.src.html]: Actually, I think it has to be the second step of the while loop. sequence[0] is already confirmed to be defined, and there is no check between incrementing by 1 in the whitespace loop and reaching the SOLIDUS loop.

Comment (Ritsyy) [inline Overview.src.html]: okay yeah got your point, so making the check over second step of the while loop.

**Comment by GPHemsley** [comment]
r+

**Comment by GPHemsley** [comment]
Merged as 0be57a9, per README.

**Comment by Ritsyy** [comment]
@GPHemsley Thanks!

**Pull request merged**